docs/src/pages/components/data-grid/components/components.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Data Grid - Components
+components: DataGrid, XGrid
+---
+
+# Data Grid - Components
+
+<p class="description">The grid is highly customizable. Override components using the `components` prop.</p>
```

Comment:
` is not supported

```suggestion
<p class="description">The grid is highly customizable. Override components using the <code>components</code> prop.</p>
```

or

```suggestion
<p class="description">The grid is highly customizable. Override components using the components prop.</p>
```

Reply:
>` is not supported 

why? it seems to work

Reply:
It renders broken. 

<img width="962" alt="Screenshot 2021-04-11 at 18 31 09" src="https://user-images.githubusercontent.com/3165635/114312768-1af69f00-9af4-11eb-9fa4-c6768a37e2f2.png">

Reply:
I see! The `tick` notation doesn't work in the description.